pkg/dashboards/kubernetes/kubelet/kubelet.go

```diff
@@ -0,0 +1,167 @@
+package kubelet
+
+import (
+	"github.com/perses/community-dashboards/pkg/dashboards"
+	panelsGostats "github.com/perses/community-dashboards/pkg/panels/gostats"
+	panels "github.com/perses/community-dashboards/pkg/panels/kubernetes"
+	"github.com/perses/community-dashboards/pkg/promql"
+	"github.com/perses/perses/go-sdk/dashboard"
+	panelgroup "github.com/perses/perses/go-sdk/panel-group"
+	labelValuesVar "github.com/perses/perses/go-sdk/prometheus/variable/label-values"
+	listVar "github.com/perses/perses/go-sdk/variable/list-variable"
+)
+
+func withKubeletStats(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Kubelet Stats",
+		panelgroup.PanelsPerLine(6),
+		panels.RunningKubeletStat(datasource, labelMatcher),
+		panels.RunningPodStat(datasource, labelMatcher),
+		panels.RunningContainersStat(datasource, labelMatcher),
+		panels.ActVolumeCountStat(datasource, labelMatcher),
+		panels.DesiredVolumeCountStat(datasource, labelMatcher),
+		panels.ConfigErrorCountStat(datasource, labelMatcher),
+	)
+}
+
+func withKubeletOperations(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Operation Rate and Errors",
+		panelgroup.PanelsPerLine(2),
+		panels.RunningKubeletStat(datasource, labelMatcher),
+		panels.RunningPodStat(datasource, labelMatcher),
```

Comment:
I think these should be, 
```suggestion
		panels.OperationRate(datasource, labelMatcher),
		panels.OperationErrorRate(datasource, labelMatcher),
```

Reply:
Right now it shows 
![Screenshot 2025-05-12 at 08 30 24](https://github.com/user-attachments/assets/cda92ece-bb78-4b6f-b76c-fd98194e945b)